### ISSUE TITLE
Display social media feed to boost user engagement, build credibility, maximize conversion, and more.

### ISSUE THREAD

**Issue by huenique** [SurPathHub/events#29]
**Description**
Consider embedding social media feed(s) showing member and community related activities on sites like Facebook, LinkedIn, Twitter, and Instagram. Embedding social media plugins to a webpage has numerous advantages justified by countless well-researched articles with credible citations. Some are included in the title of this issue.

**Describe the solution you'd like**
A component showing recent posts from social media users using community/community event related hashtags, a.k.a. embedded social media hashtag feed.

**Describe alternatives you've considered**
It's currently not feasible, if not impossible, to have a hashtag feed for every social media platform due to service and API limitations. Applying workarounds aren't encouraged because of legalities.

Alternatively, we can embed the community's social media page(s) feed/timeline if the hashtag feed idea isn't possible altogether, but I believe this should be on the homepage.

**Comment by vinzvinci** [comment]
Good idea 👍, but needs more research, we can't integrate this into current events. We can use this feature for our next event if needed. I'll put this `on hold`. Love to hear more from your views.
